plugins/chat/assets/javascripts/discourse/services/chat-state-manager.js

```diff
@@ -166,12 +167,16 @@ export default class ChatStateManager extends Service {
   }
 
   storeAppURL(url = null) {
-    if (url) {
-      this._appURL = url;
-    } else if (this.router.currentURL?.startsWith("/chat")) {
-      this._appURL = "/";
+    const urlToStore = url || this.router.currentURL;
+
+    if (
+      urlToStore &&
+      !urlToStore.startsWith("/chat") &&
+      !urlToStore.startsWith("/discourse-ai/ai-bot")
```

Comment:
The chat plugin should ideally not contain any concerns of another plugin. I can't think of a way to solve this easily from the technical perspective. Maybe we just leave with the fact that we will have `AI -> Chat -> AI` or `Chat -> AI -> Chat`?